js/private/js_image_layer.bzl

```diff
@@ -0,0 +1,140 @@
+"creates tar layers from js_binary targets"
```

Comment:
this docstring appears at the top of the markdown, so it needs more text and capitalization, etc